style.css

```diff
@@ -173,20 +174,21 @@ a {
 
 .project-container {
```

Comment:
- [ ] As requestion by the previous reviewer about using the CSS grid, which well implemented in this section so far, however fun fact about `grid` it doesn't do too much by itself, so to achieve aligning the projects items kindly specify a template for the grid, as an example:

```css
    grid-template: 1fr repeat(4, 1fr);
```

- for more information please follow [https://css-tricks.com/snippets/css/complete-guide-grid/](https://css-tricks.com/snippets/css/complete-guide-grid/)